src/app/data/signatures.ts

```diff
@@ -81,7 +81,7 @@ function getMoveLearnableGroups(): MoveCounts {
     });
 
     Object.values(pokemon).forEach((speciesData) => {
-        if (speciesData.getEvos().length > 0) return;
+        if (speciesData.isFinalEvo()) return;
```

Comment:
This (and its cousin 40 lines up) should be negated, the way Tectonic assigns signatures is to *only* final evos